src/pages/NewBeer.js

```diff
@@ -0,0 +1,83 @@
+import { Button, Col, Divider, Input, Row } from "antd";
+import React from "react";
+import { useNavigate } from "react-router-dom";
+import { createNewBeer } from "../service/services";
+
+function NewBeer() {
+  const navigate = useNavigate();
+  async function createBeer(e) {
+    e.preventDefault();
+
```

Comment:
what if the form is about 50 inputs ? this is not dinamic has to have handle change